robotd/game_specific.py

```diff
@@ -0,0 +1,18 @@
+"""Game specific values."""
```

Comment:
I think this file has been committed accidentally. This file was moved to `sb-vision`